lucien-ios/LoginScreen/LoginViewModel.swift

```diff
@@ -49,4 +49,74 @@ final class LoginViewModel {
             }
         }
     }
+
+    // TODO: Replace completion (Error?) with completion (Comic?) when Comic model is complete.
+    func addComicBook(comicTitle: String,
+                      storyTitle: String,
+                      volume: String?,
+                      issueNumber: String?,
+                      publisher: String?,
+                      releaseYear: String?,
+                      comicPhotoURL: String?,
+                      returnDate: String?,
+                      condition: String?,
+                      genre: String?,
+                      completion: @escaping (Error?) -> Void) {
+
+        lucienAPIClient.addComicBook(comicTitle: comicTitle,
+                                     storyTitle: storyTitle,
+                                     volume: volume,
+                                     issueNumber: issueNumber,
+                                     publisher: publisher,
+                                     releaseYear: releaseYear,
+                                     comicPhotoURL: comicPhotoURL,
+                                     returnDate: returnDate,
+                                     condition: condition,
+                                     genre: genre) { response in
```

Comment:
the `) { response in` should be on its own line indented at the same level as `lucienAPIClient`